code/modules/power/tesla/energy_ball.dm

```diff
@@ -54,6 +54,20 @@ GLOBAL_LIST_INIT(blacklisted_tesla_types, typecacheof(list(/obj/machinery/atmosp
 
 	. = ..()
 
+/obj/singularity/energy_ball/admin_investigate_setup()
```

Comment:
You can change this entire proc to be
```
if(istype(loc, /obj/singularity/energy_ball)
    return
. = ..()
```
The parent proc already does everything you need, you just need to ignore when the ball is orbiting the other.